src/test/java/com/google/sps/data/CommentTest.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+import static org.junit.Assert.assertTrue;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Representative;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CommentTest {
```

Comment:
you probably don't need this test. it does nothing. also see other comment about why toxicity is in the PR.

Reply:
I'll remove